version.json

```diff
@@ -1 +1,4 @@
-{"versionName":"2021.3.4","versionCode":100042155}
\ No newline at end of file
+{
+  "versionName": "2021.3.4",
+  "versionCode": 100042155
+}
```

Comment:
Revert this change please, probably because of prettier, right? I added it to ignored files somewhere already, not sure in which PR though.